src/pages/ProjectShowcasePage/ProjectShowcasePage.tsx

```diff
@@ -50,51 +22,113 @@ const ProjectShowcasePage = () => {
+                    </div>
+                    <PDFViewer pdfFileName={pdfFiles} />
+                    <div className='flex justify-evenly my-[2rem]'>
+                        {projectFiles?.map((projectFile: string, index: number) => 
+                            <DownloadWithFilename key={index} filename={getFileNameFromUrl(projectFile)} ref={el => downloadRefs.current[index] = el} />
+                        )}
+                    </div>
+                    <h2 className="my-[0.6rem] text-MVP-black text-[1.5rem] font-extrabold leading-[130.645%] font-gilroy ligature-off">Tech Stack: *</h2>
+                    <ul>
+                        {techStack?.split(",").map(item => item.trim()).map((tech, index) => (
+                            <li key={index} className="text-MVP-black font-normal text-[1.3rem] leading-[130.645%] font-poppins my-[0.2rem]">
+                                {tech}
+                            </li>
+                        ))}
+                    </ul>
+                    <h2 className="my-[1.3rem] text-MVP-black text-[1.5rem] font-extrabold leading-[115.645%] font-gilroy ligature-off">Design Tool Used: *</h2>
+                    <ul>
+                        {designTools?.split(",").map(item => item.trim()).map((tool, index) => (
+                            <li key={index} className="text-MVP-black font-normal text-[1.3rem] leading-[115.645%] font-poppins my-[0.2rem]">
+                                {tool}
+                            </li>
+                        ))}
+                    </ul>
+                    <h2 className="my-[1.3rem] text-MVP-black text-[1.5rem] font-extrabold leading-[115.645%] font-gilroy ligature-off">Project Links:</h2>
+                    <ul>
+                        {projectLinks?.map(({url}, index) => (
+                            <li key={index} className="text-MVP-black font-normal text-[1.3rem] leading-[115.645%] font-poppins my-[0.4rem] hover:underline">
+                                <Link to={url}>
+                                    {url}
+                                </Link>
+                            </li>
+                        ))}
+                    </ul>
+                    <HorizontalTabMenu 
+                        tabs={[
+                            {id: 'problemStatement', label: 'Problem Statement', content: problemStatement },
+                            {id: 'projectChallenge', label: 'Project Challenge', content: basicProjectSummary },
+                            {id: 'designFeatures', label: 'Design Features', content: designFeatures },
+                            {id: 'nextSteps', label: 'Next Steps', content: nextSteps}
+                        ]}
+                    />
+                    <h1 className="flex-1 text-MVP-black font-gilroy text-[1.5rem] font-extrabold leading-[1.8rem]">
```

Comment:
It's important to use heading tags (<h1>, <h2>, <h3>, etc.) in a logical order. Each page should only have one <h1>, and subsequent headings (<h2>, <h3>, etc.) should follow in a hierarchical sequence. For example, you shouldn't use an <h3> unless you've already used an <h2>. This structure helps search engines and assistive technologies understand the content better, improving both SEO and accessibility.